app/views/capture-debt.njk

```diff
@@ -0,0 +1,145 @@
+{% extends '_layout.njk' %}
```

Comment:
It would be good to incorporate the error messages on to each of the form components:

https://design-system.service.gov.uk/components/error-message/